app/routes/photos/components/GalleryPictureModal.module.css

```diff
@@ -1,14 +1,27 @@
 @import url('~app/styles/variables.css');
 
+.loadingIndicator {
+  margin: 0.375rem;
```

Comment:
no

Reply:
It should match the padding of clickable `Icon`s: https://github.com/webkom/lego-webapp/blame/8d156d0615da926ca9b2d0cc12037576fa4bc223/packages/lego-bricks/src/components/Icon/Icon.module.css#L24

Reply:
It avoids a slight movement in the  layout, which doesn't look great.